app/src/main/java/com/pluu/support/naver/NaverWeekApi.kt

```diff
@@ -40,8 +40,8 @@ class NaverWeekApi(
                         title = element.select(".toon_name").text()
                         image = element.select("img").first().attr("src")
                         status = when {
-                            element.select("em[class=badge badge_up]").isNotEmpty() -> Status.UPDATE  // 최근 업데이트
-                            element.select("em[class=badge badge_break]").isNotEmpty() -> Status.BREAK  // 휴재
+                            element.select("em[class=badge badge_up]").isNotEmpty() -> Status.UPDATE // 최근 업데이트
+                            element.select("em[class=badge badge_break]").isNotEmpty() -> Status.BREAK // 휴재
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/3998f0ffc98de81a3e6a71a93261f4a2d285e04c/app/src/main/java/com/pluu/support/naver/NaverWeekApi.kt#L44"></span>Exceeded max line length (100) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>